RocketChat/Rocket.Chat
# [NEW] Open modals in side effects outside React

## Proposed changes (including videos or screenshots)





## Issue(s)


## Steps to test or reproduce

- Import `imperativeModal` from `client/lib/imperativeModal.ts` and call `imperativeModal.open({ component, props })` somewhere. It should open the provided modal in the React Context. `imperativeModal.close()` should close it.

## Further comments
